tests81/Unit/CrudPanel/CrudPanelUpdateTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Backpack\CRUD\Tests81\Unit\CrudPanel;
+
+use Backpack\CRUD\Tests\Unit\CrudPanel\BaseDBCrudPanelTest;
+use Faker\Factory;
+
+/**
+ * @covers Backpack\CRUD\app\Library\CrudPanel\Traits\Update
+ */
+class CrudPanelUpdateTest extends BaseDBCrudPanelTest
+{
+    public function testGetUpdateFieldsWithEnum()
+    {
+        if ((int) app()->version() < 9) {
```

Comment:
```suggestion
        if ((int) app()->version() < 8) {
```

Enum casting is available in Laravel 8 too

https://laravel.com/docs/8.x/eloquent-mutators#enum-casting